python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -638,6 +676,15 @@ class carrying all of the information for the object whose image
         gs_sed = galsim.SED(sed_lut, wave_type='nm', flux_type='flambda',
                             redshift=0.)
 
+        local_hour_angle \
+            = self.getHourAngle(self.obs_metadata.mjd.TAI,
+                                self.obs_metadata.pointingRA)/galsim.degrees
```

Comment:
Did this actually work?  It should be `*`, not `/`.  You have a bunch of `/galsim.degrees` here where I think there should be `*galsim.degrees`.  I see unit tests which I would have thought would catch this, but I notice that Travis isn't running them, so maybe they haven't actually been run?

And of course, given my above comment, you don't need either here, since you can just get ha as an Angle already from `getHourAngle`.